src/DotNetStatus/JsonData/JsonData.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fx.Portability
+{
+    public class JsonData
+    {
+        private HttpClient _client;
```

Comment:
Can you model this class off of the ApiPortService - this could be called ApiPortStatusService or something. It would be best to include all methods to retrieve information instead of passing in a query